carl/envs/rna/data/download_and_build_rfam_learn.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# mkdir -p data/rfam_learn/{raw,test,train,validation}
```

Comment:
mkdir not needed?

Reply:
what about windows systems? is there a comment that RNA only works on linux?

Reply:
Done